backend/de.metas.acct.base/src/main/sql/postgresql/ddl/functions/AccountSheet_Report.sql

```diff
@@ -0,0 +1,56 @@
+DROP FUNCTION IF EXISTS report.AccountSheet_Report(
+    p_Account_ID      numeric,
+    p_C_AcctSchema_ID numeric,
+    p_DateAcctFrom    DATE,
+    p_DateAcctTo      date
+)
+;
+
+
+CREATE OR REPLACE FUNCTION report.AccountSheet_Report(
+    p_Account_ID      numeric,
+    p_C_AcctSchema_ID numeric,
+    p_DateAcctFrom    DATE,
+    p_DateAcctTo      date
+)
+    RETURNS
+        TABLE
+        (
+            konto         text,
```

Comment:
since this is part of metasfresh repo, wondering why aren't we use nice column names which will be translated via AD_Elements?

Reply:
gonna do that in next step; still work in progress